cns/service/main.go

```diff
@@ -856,20 +858,32 @@ func main() {
 			logger.Errorf("Failed to start multiTenantController, err:%v.\n", err)
 			return
 		}
+	}
 
+	logger.Printf("[Azure CNS] Start HTTP local echo server")
+	httpEchoRestService := restserver2.New(httpRestService)
+	if httpEchoRestService != nil {
+		go func() {
```

Comment:
im concerned about putting these servers' `Start` func within goroutines without passing in a context to prevent leaks.

Reply:
I would share those concerns, @nddq

Reply:
yes we should add context to control this async